policyengine_us/variables/gov/states/ar/tax/income/exemptions/ar_retirement_or_disability_benefits_exemption_person.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class ar_retirement_or_disability_benefits_exemption_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Arkansas individual retirement or disability benefits exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=13"
+    defined_for = StateCode.AR
+
+    def formula(person, period, parameters):
```

Comment:
should disability benefits be included here?

Reply:
Yes, I'll add it.